source/appendices/f-handling-errors-and-exceptions.rst

```diff
@@ -148,6 +148,12 @@ Table F.1 (below): Exceptions
      - 403
      - If requestor_id has not been authorized to harvest usage for the institution identified by the customer_id, or if the customer_id is not recognized.
 
+   * - Global Reports Not Supported
+     - Error
+     - 2011
+     - 403
+     - Reporting to "The World", customer_id 0000000000000000, is not supported.
+
```

Comment:
If we define a separate Exception for this case, content providers that don't support global reports would be required to support this Exception. This would be fine with me, but I'm not sure this was intended?

Reply:
If we don't include an exception I suspect we'll get a lot of questions from report consumers